test/data/v1beta1/build_buildah_cr_local_source_upload.yaml

```diff
@@ -5,9 +5,9 @@ metadata:
   name: buildah-golang-build-local-source-upload
 spec:
   source:
```

Comment:
The typical use case for typ: Local is that one has a Git source in the Build, but overwrites it in the BuildRun. Having said this, can you undo the changes in this file and specify the source section in the BuildRun. It should work assuming we correctly defined the conversion logic.